finality-aleph/src/network/tests.rs

```diff
@@ -235,178 +252,182 @@ async fn test_network_event_sync_disconnected() {
-                assert_eq!(protocol, PROTOCOL_NAME);
-                match <(SessionId, Vec<u8>) as Decode>::decode(&mut message.as_slice()) {
-                    Ok((session_id_, message)) => {
-                        assert_eq!(session_id_.0, 0);
-                        assert_eq!(message, fake_message);
-                    }
-                    _ => panic!("Expected a properly encoded message"),
-                }
-            }
-            _ => panic!("Expecting a message"),
-        }
-    }
-
-    println!("send to all");
-
-    // SendToAll
-    {
-        let fake_message: Vec<u8> = vec![205];
-        data.rush_network
-            .net_command_tx
-            .send(SessionCommand {
-                session_id: data.rush_network.session_id,
-                command: NetworkCommand::SendToAll(fake_message.clone()),
-            })
-            .await
-            .unwrap();
-        let mut peer_ids = HashSet::<PeerId>::new();
-        for _ in 0..2_u8 {
-            match data.network.send_message.1.lock().next().await {
-                Some((peer_id, protocol, message)) => {
-                    peer_ids.insert(peer_id);
-                    assert_eq!(protocol, PROTOCOL_NAME);
-                    match <(SessionId, Vec<u8>)>::decode(&mut message.as_slice()) {
-                        Ok((session_id, message)) => {
-                            assert_eq!(session_id.0, 0);
-                            assert_eq!(message, fake_message);
-                        }
-                        _ => panic!("Expected a properly encoded message"),
-                    }
-                }
-                _ => panic!("Expected two messages"),
-            }
-        }
-        let expected_peer_ids: HashSet<_> = [data.bob.peer_id, data.charlie.peer_id]
-            .iter()
-            .cloned()
-            .collect();
-        assert_eq!(peer_ids, expected_peer_ids);
-    }
-
-    // SendToRandPeer
-    {
-        let fake_message = vec![74];
-        data.rush_network
-            .net_command_tx
-            .send(SessionCommand {
-                session_id: data.rush_network.session_id,
-                command: NetworkCommand::SendToRandPeer(fake_message.clone()),
-            })
-            .await
-            .unwrap();
-        match data.network.send_message.1.lock().next().await {
-            Some((peer_id, protocol, message)) => {
-                assert!(peer_id == data.bob.peer_id || peer_id == data.charlie.peer_id);
-                assert_eq!(protocol, PROTOCOL_NAME);
-                match <(SessionId, Vec<u8>)>::decode(&mut message.as_slice()) {
-                    Ok((session_id, message)) => {
-                        assert_eq!(session_id.0, 0);
-                        assert_eq!(message, fake_message);
-                    }
-                    _ => panic!("Expected a properly encoded message"),
+    // Wait for acknowledgement that Alice noted Bob's presence.
+    let _ = data
```

Comment:
Isn't `let _ = ` pointless here and below?

Reply:
Yup, must have been some leftovers, thanks!